op-challenger/game/fault/register.go

```diff
@@ -254,7 +260,7 @@ func registerAsterisc(
 			if err != nil {
 				return nil, fmt.Errorf("failed to get asterisc prestate: %w", err)
 			}
-			accessor, err := outputs.NewOutputAsteriscTraceAccessor(logger, m, cfg.Asterisc, l2Client, prestateProvider, asteriscPrestate, rollupClient, dir, l1HeadID, splitDepth, prestateBlock, poststateBlock)
+			accessor, err := outputs.NewOutputAsteriscTraceAccessor(logger, m, cfg.Asterisc, l2Client, prestateProvider, asteriscPrestate, rollupClient, dir, l1HeadID, splitDepth, prestateBlock, poststateBlock, serverType)
```

Comment:
The `cfg.Asterisc` being passed here is setup to have the `Server` option point to `op-program`.  We'd need a separate CLI flag to be able to specify a path to kona instead of op-program.  Possibly rather than `--cannon-server` and `--asterisc-server` we need `--kona` and `--opprogram` options or something like that. We'll also need new options to specify the `asterisc/kona` prestate.